pkg/operator/ceph/object/objectstore.go

```diff
@@ -351,3 +354,76 @@ func GetObjectBucketProvisioner(c *clusterd.Context, namespace string) string {
 	}
 	return provName
 }
+
+func checkDashboardUser(context *Context) bool {
+	args := []string{"dashboard", "get-rgw-api-access-key"}
+	cephCmd := ceph.NewCephCommand(context.Context, context.ClusterName, args)
+	out, _ := cephCmd.Run()
+	if string(out) == "" {
+		return false
+	} else {
+		return true
+	}
+}
+
+func enableRGWDashboard(context *Context) error {
+	if checkDashboardUser(context) {
+		logger.Debug("Dash board already enabled for rgw")
+		return nil
+	}
+	//currently adminuser only for accessing dashboard, in future this will be seperate api
+	dispname := fmt.Sprintf("%s", AdminUser)
+	user := ObjectUser{
+		UserID:      AdminUser,
+		DisplayName: &dispname,
+		SystemUser:  true,
+	}
+	u, errCode, err := CreateUser(context, user)
+	if err != nil || errCode != 0 {
+		return errors.Wrap(err, "error creating admin user")
```

Comment:
More clear, especially if the user (unlikely) changes in the future, but this is cleaner code.

```suggestion
		return errors.Wrap(err, "error creating %q user", AdminUser)
```